docs/usage.md

```diff
@@ -92,3 +100,7 @@ Nextflow handles job submissions and supervises running jobs. The Nextflow proce
 #### `--platform`
 
 This argument is required and you must input either `illumina` or `nanopore` depending on your input sequencing files. Therefore a combination of Illumina and Nanopore sequencing files **CANNOT** be run in the same submission - instead submit two jobs differentiating between the two file types.
+
+#### `--sourmash_dbs`
+
+This argument is required and you must download the sourmash databases and lineage CSVs prior to running the workflow. You can find several pre-prepared sourmash databases at [https://sourmash.readthedocs.io/en/latest/databases.html](https://sourmash.readthedocs.io/en/latest/databases.html). It is highly recommended that for the purposes of this workflow you download the same kmer size for each database, and we use `k31` as a default kmer size throughout the workflow. Once you have these downloaded, prepare the CSV as shown above.
```

Comment:
It's not even a recommendation, you can't mix k-mer sizes within a single sourmash gather run

Reply:
addressed